Purchases/Networking/RCBackend.m

```diff
@@ -414,9 +425,21 @@ - (void)handleLoginWithResponse:(nullable NSDictionary *)response
 
 - (void)createAliasForAppUserID:(NSString *)appUserID
                withNewAppUserID:(NSString *)newAppUserID
-                     completion:(nullable void (^)(NSError * _Nullable error))completion {
+                     completion:(nullable PostRequestResponseHandler)completion {
     NSString *escapedAppUserID = [self escapedAppUserID:appUserID];
+    if (!escapedAppUserID || [escapedAppUserID isEqualToString:@""]) {
```

Comment:
This if is great, thanks for adding it. Two comments. Can you also check `newAppUserID` is empty or nil? I am not sure if it's done somewhere else. The other thing is if you can extract the log message to the Logger strings it would be great

Reply:
so looking at the code, logIn actually uses: 

    NSParameterAssert(currentAppUserID);
    NSParameterAssert(newAppUserID);

i actually kinda prefer our own error messages for providing more detail, but sounds like NSParameterAssert will cause a crash...not sure which is preferable. either way, i feel like we should be consistent...thoughts?

Reply:
It will cause a crash in `DEBUG`, but currently, it won't crash in `RELEASE`  (you can check for `Enable Foundation Assertions` in the project setting)
I think we should do both- assert and also nice error messages 😄